log.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"os"
+	"regexp"
+)
+
+const (
+	// Severity.
+
+	// From /usr/include/sys/syslog.h.
+	// These are the same on Linux, BSD, and OS X.
+	LOG_EMERG int = iota
+	LOG_ALERT
+	LOG_CRIT
+	LOG_ERR
+	LOG_WARNING
+	LOG_NOTICE
+	LOG_INFO
+	LOG_DEBUG
+)
+
+const (
+	EnvLogLevel      = "LOG_LEVEL"
+	EnvLogFormat     = "LOG_FORMAT"
+	defaultLogFormat = "<%s>\t%s" // #1 severity, #2 msg
```

Comment:
Double checked that. Journald does not take <ERR>, but takes <2>. also we don' want a tab when logging to systemd.